main.go

```diff
@@ -140,6 +140,7 @@ func validateOptions(cmd *cobra.Command) error {
 	width = viper.GetUint("width")
 	localOnly = viper.GetBool("local")
 	mouse = viper.GetBool("mouse")
+	pager = viper.GetBool("pager")
```

Comment:
I wonder if we should make this a `bool` or let the user specify the pager command as a string here. As it's currently in-line with how the `--pager` flag works, I think this is already an improvement I'll merge, though. Open for discussion nonetheless.